index.html

```diff
@@ -329,17 +329,17 @@ <h2 class="connection m-0 p-l-15">
 
             </div>
         </section>
-        <section class="contact w-100 b-top-left-rounded">
+        <div class="contact w-100 b-top-left-rounded">
             <h2 class="w-100">Contact me</h2>
             <div class=" contact-text w-90 p-15 m-auto">
                 <p>
                     If you have an application you are interested in developing,
                  a feature that you need built or a project that needs coding. I’d love to help with it
                 </p>
                 <form class="contact-form w-100 m-auto">
-                    <input type="text" class="w-90" placeholder=" Your name" id="name"/>
-                    <input type="text" class="w-90" placeholder="natajayanj14@gmail.com" id="email"/>
-                    <textarea class="w-90">
+                    <input type="text" required class="w-90" placeholder=" Your name" id="name"/>
+                    <input type="text" required class="w-90" placeholder="natajayanj14@gmail.com" id="email"/>
+                    <textarea class="w-90" required>
                         Write your message here
```

Comment:
with Textarea field should put a max length attribute for HTML validation

Reply:
- [ ] Kindly make your comments free from grammatical and other sentence structure mistakes.

Reply:
- This issue is well addressed. Nice work! :clap: